includes/theme-compat/register-profile.php

```diff
@@ -0,0 +1,104 @@
+<section class="section-fullwidth">
+    <div class="row">
+        <div class="flex-container centered-vertically centered-horizontally">
+            <div class="form-box box-shadow">
+                <div class="section-heading">
+                    <?php if (!empty($page_name) && $page_name == 'profile') { ?> <h2 class="heading-title">My
+                        Profile</h2> <?php } ?>
+                    <?php if (!empty($page_name) && $page_name == 'register') { ?> <h2 class="heading-title">
+                        Register</h2> <?php } ?>
+                </div>
+                <form action="/form.php" id="register" method="post" enctype="multipart/form-data">
+                    <div class="flex-container justified-horizontally">
+                        <div class="primary-container">
+                            <h4 class="form-title">About me</h4>
+                            <input type="hidden" name="form_name" value="register"/>
+                            <div class="form-field-wrapper">
+                                <input type="text" name="first_name" placeholder="First Name*"
+                                       <?php if (!empty($_SESSION['first_name'])) {
+                                           echo 'value="' . $_SESSION['first_name'] . '" style="color: #3c71fe;" ';
```

Comment:
inline styling alert, we should avoid inline styling. The same comment would be applicable in a few other places in this file

Reply:
I moved the inline styling to css file